.changeset/stale-mugs-clean.md

```diff
@@ -0,0 +1,17 @@
+---
+'@commercetools-applications/merchant-center-custom-view-template-starter-typescript': minor
+'@commercetools-applications/merchant-center-template-starter-typescript': minor
+'@commercetools-frontend/application-components': minor
+'@commercetools-applications/merchant-center-custom-view-template-starter': minor
+'@commercetools-applications/merchant-center-template-starter': minor
+'@commercetools-website/components-playground': minor
+'@commercetools-frontend/react-notifications': minor
+'@commercetools-frontend/application-shell': minor
+'@commercetools-website/custom-views': minor
+'@commercetools-frontend/constants': minor
+'@commercetools-local/visual-testing-app': minor
+'@commercetools-frontend/i18n': minor
+'@commercetools-local/playground': minor
+---
+
+Implement updated color scheme across all App-kit components
```

Comment:
@emmenko @ddouglasz What do you think about this?

Reply:
IMO, an update in this context may not necessarily mean a complete change/override. It could still mean provision for the newly supported colour scheme. But I see your  point, it sounds more explicit to say we now support more colour schemes.

Reply:
Thank you all. I'm okay with the changes, let me push the new version